### DISCUSSION TITLE
Problems after installing .rb file manually

### DISCUSSION THREAD

**Discussion by circumeo** [Homebrew/discussions#3958] [Casks]
`brew install --cask calibre.rb`

I installed the old version as a .rb file through the command I wrote above.

By the way, the problem is that the brew-outdated output includes calibre, and if you issue the brew upgrade command, the calibre will be upgraded. Is there any way to prevent the calibre from being upgraded?

Thank you in advance 🙏

<img width="600" alt="screenshot_2022-11-23_17-38-47" src="https://user-images.githubusercontent.com/76243664/203502903-f00bf341-9a23-44e5-93dc-c4a40e2d699c.png">
<img width="600" alt="screenshot_2022-11-23_17-39-02" src="https://user-images.githubusercontent.com/76243664/203502917-4ed64907-a3fb-4fb7-a31a-e90207c0c99a.png">
<img width="600" alt="screenshot_2022-11-23_17-45-29" src="https://user-images.githubusercontent.com/76243664/203503842-1af76317-a762-40c3-afa8-ed42987c99f1.png">

**Comment by SMillerDev** [comment]
There is not, unless you maintain the older version in a tap.

**Comment by p-linnane** [comment]
You can't pin casks as far as I'm aware. You can either host the old version [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap), or just install it manually outside `brew`.
